bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/api/DoorbirdAPI.java

```diff
@@ -102,6 +102,21 @@ public void setHttpClient(HttpClient httpClient) {
         return doorbirdInfo;
     }
 
+    public @Nullable DoorbirdSession getSession() {
+        DoorbirdSession doorbirdSession = null;
+        try {
+            String sessionResponse = executeGetRequest("/bha-api/getsession.cgi");
+            doorbirdSession = new DoorbirdSession(sessionResponse);
+        } catch (IOException e) {
+            logger.info("Unable to communicate with Doorbird: {}", e.getMessage());
+        } catch (JsonSyntaxException e) {
+            logger.info("Unable to parse Doorbird response: {}", e.getMessage());
```

Comment:
```suggestion
            logger.warn("Unable to parse Doorbird response: {}", e.getMessage());
```
Don't you also want to set the thing status to offline communication eror? If so, it can be set to debug, as it provides some more detail then the thing status

Reply:
See my comment above.